modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -62,6 +63,9 @@
     /** Table identifier. */
     private UUID tableId;
 
+    /** Table schema mode. May be STRICT or LIVE */
+    private SchemaMode schemaMode;
```

Comment:
```suggestion
    private volatile SchemaMode schemaMode;
```
Does this make sense?